packages/contracts/contracts/optimistic-ethereum/OVM/execution/OVM_ExecutionManager.sol

```diff
@@ -1920,9 +2129,39 @@ contract OVM_ExecutionManager is iOVM_ExecutionManager, Lib_AddressResolver {
             (bool success, bytes memory returndata) = ovmCALL(
                 _transaction.gasLimit,
                 _transaction.entrypoint,
+                _value,
                 _transaction.data
             );
             return abi.encode(success, returndata);
         }
     }
+
+
+    /**
+     * Unreachable helper function for setting an account's balance so that eth_calls with value always work.  Only used by simulateMessage().
+     * @param _of the OVM account to set the balance of.
+     * @param _balance the desired balance for the account.
+     */
+    function _setOvmEthBalance(
+        address _of,
+        uint256 _balance
+    )
+        internal
+    {
+        // Get the correct storage slot for OVM_ETH's balanceOf[_of] slot
+        uint256 balanceOfStorageSlot = 5;
```

Comment:
Entertaining that this works but it seems potentially dangerous if we ever make a change to `OVM_ETH` and forget about it later on. Is there a simpler way to do this? Or at least a way that will break if we ever make a change to `OVM_ETH`?

Reply:
Instead of exposing a forced transfer maybe we could expose a forced "set balance"... not sure if that's better though. Maybe it's best to just have both functions so that this can be set more directly.

Reply:
I could not think of a cleaner way to do this, other than triggering an `OVM_ETH.mint` by somehow faking a deposit.  Considering that if the `OVM_ETH` storagelayout changes, we will already have to make updates e.g. to geth's stateManager, I took this to be good enough.  LMK if you agree.

Reply:
see above; this code got removed--EVM doesn't do this anyway.